crates/js_api/src/gui/order_operations.rs

```diff
@@ -467,30 +467,27 @@ impl DotrainOrderGui {
 
         let mut approvals = Vec::new();
         let approval_calldata = self.generate_approval_calldatas(owner).await?;
-        match approval_calldata {
-            ApprovalCalldataResult::Calldatas(calldatas) => {
-                let mut output_token_infos = HashMap::new();
-                for output in deployment.deployment.order.outputs.clone() {
-                    if output.token.is_none() {
-                        return Err(GuiError::SelectTokensNotSet);
-                    }
-                    let token = output.token.as_ref().unwrap();
-                    let token_info = self.get_token_info(token.key.clone()).await?;
-                    output_token_infos.insert(token.address.clone(), token_info);
+        if let ApprovalCalldataResult::Calldatas(calldatas) = approval_calldata {
```

Comment:
were these modifications required for this PR? i can't see any tests for this file, it would be better if we are making updates that we have accompanying tests, otherwise i get nervous we're introducing a bug